regression/scripts/AVL/avl_files/aircraft.avl

```diff
@@ -62,7 +62,7 @@ SECTION
 
 SECTION
 #Xle    Yle      Zle      Chord     Ainc  Nspanwise  Sspace
-32.4164746575  1.54    7.777    0.95    0.0     
+32.4164746575  0.0    7.777    0.95    0.0     
```

Comment:
??? @mclarke2 needs to look at this....

Reply:
Yeah, I think something weird might have happened merging the latest version of develop. I'll use the old values unless @mclarke2  wants to stick with the newer values

Reply:
It is a vertical stabilizer, the y value should be 0 so I corrected it.